.vale/styles/Google/WordList.yml

```
@@ -61,6 +61,7 @@ swap:
   in order to: to
   ingest: import|load
   k8s: Kubernetes
+  "[kK]ube[rR]ray": KubeRay
```

I think Vale gets confused if we do `"[kK]ube[rR]ray"` because it includes the correct spelling ("KubeRay")
```suggestion
  "[Kk]uberay": KubeRay
```